src/router.js

```diff
@@ -149,7 +149,7 @@ const routes = [
   {
     path: '/networking',
     name: 'networking',
-    component: Networking,
+    component: Conference,
     meta: {
       title: 'Networking',
     },
```

Comment:
Can you combine this route with the next one?

Reply:
unfortunately aliases does not work with params. But I will try to add `networking` alias to the `conference` maybe

Reply:
it also doesn't work with `alias: [ '/networking' ],` and `<router-link :to="{ name: 'conference' }">`

Reply:
My idea was to have a single url with an optional `room` parameter, like:
```
{
    path: '/league/:room?',
    name: 'conference',
    component: Conference,
    props: true,
    meta: {
      title: 'Conference',
    },
  },
```
I didn't mean to use aliases. Can it work in this way?

Reply:
I guess so. Thanks.